ide/coqide/coq.ml

```diff
@@ -611,7 +612,14 @@ let db_configd x = eval_call ~db:true (Xmlprotocol.db_configd x)
 
 let interrupt_coqtop coqtop workers =
   if coqtop.status = Busy then
-    try !interrupter (CoqTop.unixpid coqtop.handle.proc)
+    try
+      let pid = CoqTop.unixpid coqtop.handle.proc in
+      if Sys.os_type = "Win32" then begin
```

Comment:
There is also a "Cygwin" value for this. Which windows setup does the issue occur on?

https://v2.ocaml.org/api/Sys.html

Reply:
There is no Cygwin build for Coq.  Indeed, CoqIDE must be compiled using MinGW libraries.  Compiling it with Cygwin libraries gives an unusable binary.  OTOH Coq platform installs Cygwin to provide Linux commands, which I also use when necessary.  I've never seen os_type "Cygwin".